src/Storefront/Resources/views/storefront/component/line-item/element/image.html.twig

```diff
@@ -35,14 +35,23 @@
         {% endif %}
     {% endset %}
 
-    {% if productLink %}
-        <a href="{{ seoUrl('frontend.detail.page', { 'productId': lineItem.referencedId }) }}"
+    {# @deprecated productLink variable will be removed without replacement, directly set lineItemLink and lineItemModalLink #}
+    {% if productLink and not lineItemLink %}
+        {% set lineItemLink = seoUrl('frontend.detail.page', { 'productId': lineItem.referencedId }) %}
+
+        {% if controllerAction is same as('confirmPage') %}
+            {% set lineItemModalLink = path('widgets.quickview.minimal', { 'productId': lineItem.referencedId }) %}
+        {% endif %}
+    {% endif %}
+
+    {% if lineItemLink %}
+        <a href="{{ lineItemLink }}"
            class="line-item-img-link"
            title="{{ label }}"
-            {% if controllerAction is same as('confirmPage') %}
-                data-toggle="modal"
+            {% if lineItemModalLink %}
+                data-ajax-modal="modal"
```

Comment:
Good catch! 👍 Really don't know why this wasn't at least `data-bs-toggle` which would have been backwards compatible until 6.6.

Reply:
Yes, I first tried to fix it like this, but than noticed the error message, and that it tried to open the modal twice :-) I did only than read the deprecation notice :-)

But this was the only place I found `data-toggle="modal"`.

Reply:
Yeah, that was why we changed it to `data-ajax-modal="true"`. :D To no longer interfere with the default modal selector from Bootstrap. But we kept the `data-bs-toggle` possibility until 6.6 in case it is still inside translations or something. Prior to 6.5, there were also 2 click events but the error did not occur, when the Bootstrap modal target was not defined (because data-url was used with our custom JS plugin). In Bootstrap 4 nothing was happening, whereas in Bootstrap 5 it shows the error.

Reply:
I see, thanks for the in depth explanation :-) I am really looking forward to the new Bootstrap 5 JavaScript.